tools/psa/tfm/bin_utils/mbed_set_post_build_tfm.cmake

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
```

Comment:
`tools/psa` is currently not covered by the frozen tools check, so I thought it would stay. I can think of two places to relocate it to:
* `platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST/scripts` (the PSA directory)
* `targets/scripts/psa`

Didn't put it directly in `targets/TARGET_ARM_SSG/scripts` because the scripts and hook should work for any of the latest TF-M targets, and can be useful if any partner adds another one in the future.

@Patater @evedon Any thoughts on this?

Reply:
`tools/psa` is currently not covered by the frozen tools check, so I thought it would stay. I can think of two places to relocate it to:
* `platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST/scripts` (the PSA directory)
* `targets/scripts/psa`

Didn't put it directly in `targets/TARGET_ARM_SSG/scripts` because the scripts and hook should work for any of the latest TF-M targets, and can be useful if any partner adds another one in the future.

@Patater @evedon Any thoughts on this?

Reply:
I think it makes sense to keep the PSA scripts with the PSA code so I like `platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST/scripts`

Reply:
+1 for keeping the scripts with the code they belong to.

Reply:
Cool, all clear now! I this PR I'll start adding files for CLI 2 to the agreed directory. When we remove CLI 1 support and delete `tools/`, that will be the time to migrate existing scripts.

Reply:
Done